storagetransfer/storagetransfer_test.go

```diff
@@ -42,6 +45,8 @@ var s3Bucket string
 var azureContainer string
 var gcsSourceBucket string
 var gcsSinkBucket string
+var pubSubSubscriptionId string
+var sqsQueueArn string
 
 func TestMain(m *testing.M) {
 	// Initialize global vars
```

Comment:
I notice that running in Kokoro seems to be skipped in L55... can this be re-enabled? The linked issue seems to be closed. FYI @muncus